.sync.yml

```diff
@@ -10,6 +10,7 @@ Gemfile:
     - gem: github_changelog_generator
 Rakefile:
   changelog_user: puppetlabs
+  changelog_since_tag: '5.3.0'
```

Comment:
This is actually redundant.  The HISTORY.md file is formatted in such a way that this will be automatically discovered.